ns/string.n3

```diff
@@ -25,15 +25,18 @@ string:concat a owl:InverseFunctionalProperty;   # obj = list of string to strin
     vs:term_status "deprecated";
     owl:inverse string:concatenation.  # Use that instead for new designs
 
-string:concatenation a owl:FunctionalProperty;
+string:concatenation a rdf:Property, owl:FunctionalProperty;
     rdfs:definedBy <>;  
     rdfs:label "is the concatenation of the strings in";
```

Comment:
This label still suggests that the result is in subject position.
```suggestion
    rdfs:label "concatenate into";
```
I am not entirely sure of this proposal, but a change is required.

Reply:
As a label, maybe it should be simply "concatenation". If you look at the deprecated `string:concat`, it's label is "is concatenation of", so that would make more sense to me, but "concatenate into" is reasonable, too.